### ISSUE TITLE
Mystery Text Discussion: cac.txt

### ISSUE THREAD

**Issue by ebeshero** [newtfire/introDH-Hub#97]
Post your screenshots and discuss your findings about cac.txt here!

**Comment by kranzrj** [comment]
In this text file I found a lot of phrases with negative connotation, and with no knowledge about the text, leads me to believe that the event in this text doesn't go the best it can. There are also a lot of formal words like "mrs. and mr.". The two most frequent uses of "Mr." appear next to Elliot or Shepherd. There seems to be much more variety in names associated with "Mrs."
![image](https://github.com/newtfire/introDH-Hub/assets/159749654/8a9062e8-1a19-4054-97db-7a39ba0285df)
![image](https://github.com/newtfire/introDH-Hub/assets/159749654/8e494fbd-a39d-475a-9145-b609a0d4cb28)

**Comment by rilegoat** [comment]
In this text, I noticed that a lot of the most frequent 4-grams are in first-person, leading me to believe that this is either a narrative or an interview---I have a hunch it's a story. Most of these 4-grams are conversational, being phrases such as "i do not ..." or "i am ...", which I think also helps to describe the delivery of the story. 
![antconc 1](https://github.com/newtfire/introDH-Hub/assets/100009027/dc9cfaf0-1de0-48a4-971a-de0c56a2350e)

Out of curiosity, I checked the KWIC for "do not know" and found that (predictably) the vast majority of these are preceded by "I" and very rarely a different word, and also are mostly followed by "what," which leads me to believe this story is more of the narrator's report on something that had happened, or speculation about the actions of another, almost like an interrogation or account given at a court trial. 
![antconc 2](https://github.com/newtfire/introDH-Hub/assets/100009027/e4662559-ed91-42b8-8374-73eff2e17cea)
